features/windows-spell-checker.json

```diff
@@ -0,0 +1,7 @@
+{
+    "_meta": {
+        "description": "Spell checker on Windows Browser"
+    },
+    "exceptions": [],
+    "state": "enabled"
```

Comment:
I'd omit the state from the base feature description and add it to https://github.com/duckduckgo/privacy-configuration/blob/main/overrides/windows-override.json instead. That way it won't show up for other platforms as an enabled feature.

Reply:
Makes sense, thanks.